internal/cmd/postgresflex/user/create/create.go

```diff
@@ -0,0 +1,139 @@
+package create
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/stackitcloud/stackit-cli/internal/pkg/args"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/confirm"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/errors"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/examples"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/flags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/globalflags"
+	"github.com/stackitcloud/stackit-cli/internal/pkg/services/postgresflex/client"
+	postgresflexUtils "github.com/stackitcloud/stackit-cli/internal/pkg/services/postgresflex/utils"
+
+	"github.com/spf13/cobra"
+	"github.com/stackitcloud/stackit-sdk-go/services/postgresflex"
+)
+
+const (
+	instanceIdFlag = "instance-id"
+	usernameFlag   = "username"
+	rolesFlag      = "roles"
+)
+
+var (
+	rolesDefault = []string{"read"}
+)
+
+type inputModel struct {
+	*globalflags.GlobalFlagModel
+
+	InstanceId string
+	Username   *string
+	Roles      *[]string
+}
+
+func NewCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "create",
+		Short: "Create a PostgreSQL Flex user",
+		Long: fmt.Sprintf("%s\n%s\n%s\n%s",
+			"Create a PostgreSQL Flex user.",
+			"The password is only visible upon creation and cannot be retrieved later.",
+			"Alternatively, you can reset the password and access the new one by running:",
+			"  $ stackit postgresflex user reset-password --instance-id <INSTANCE_ID> --user-id <USER_ID>",
```

Comment:
```suggestion
			"  $ stackit postgresflex user reset-password <USER_ID> --instance-id <INSTANCE_ID>",
```

Probably needed for mongodbflex too

Reply:
Indeed